nbproject/project.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://www.netbeans.org/ns/project/1">
+    <type>org.netbeans.modules.php.project</type>
+    <configuration>
+        <data xmlns="http://www.netbeans.org/ns/php-project/1">
+            <name>xow/moodle-mod_quizgame</name>
+        </data>
+    </configuration>
+</project>
```

Comment:
```suggestion
```

Reply:
Looks like I missed those, should now be removed